### ISSUE TITLE
OpenVPN @ Chromebook

### ISSUE THREAD

**Issue by a1exus** [kylemanna/docker-openvpn#178]
I'm trying to follow [Set up virtual private networks (VPNs) - Chromebook Help](https://support.google.com/chromebook/answer/1282338) & I'll quote:

server certificate:
> Press the Import button and choose the X.509 certificate authority file, which is usually a file with a .pem, .der, .crt, or .p7b extension.

user certificate:
> If you have a .pfx or .p12 file to import, click the Import and Bind to Device button.

---
How does one generate these file? I do have `.ovpn` file.

**Comment by kylemanna** [comment]
Closing due to inactivity.

**Comment by a1exus** [comment]
who's inactivity? I opened up an issue, asking for help, 6 month goes by and instead of help I get this?!

**Comment by kylemanna** [comment]
Welcome to the Internet.  I'll try better to deliver flawless free software so that you can be grumpy.  Cheers.

Next time you could try troubleshooting the problem here yourself after you look up the openssl docs and what not and hope that someone with nothing but free time drops what they are doing to help you.

**Comment by a1exus** [comment]
I _did_ my homework (see my question, with reference) and I _tried_ to resolve my issue (without creating issue here), unfortunately I couldn't, hence asking for help. I'm not sure who's being inactive here, if you're the one who's being inactive, it doesn't mean everyone else is... nobody forcing you to close issue! especially if it's not resolved!

**Comment by maxromanovsky** [comment]
@a1exus Try it with an OpenVPN, w/o docker. And if you succeed, create a PR into this repo instead of shouting.

**Comment by pieterlange** [comment]
@a1exus extract the user credentials from the combined .ovpn file and save them to .pem files.

OpenVPN itself assumes some basic PKI knowledge and while the responses here might be a bit harsh, this issue is definitely out of scope for a single maintainer project like this one. (which is geared towards dockerizing openvpn, not providing openvpn support)

**Comment by a1exus** [comment]
@maxromanovsky I'm trying to understand, why issue is being closed without any attempt to resolve it and I'm NOT shouting.

**Comment by a1exus** [comment]
thank you all, as of now, I actually got more feedback with my issue, then in last 6 month)

@kylemanna - this project still great and i love it, I just wish I could get that OpenVPN to go with my Chromebook, sorry if I came off bad (certainly did not mean it) and BIG thanks for your time and effort, I really appreciate it!
@maxromanovsky - i guess that's where I'm lacking.
@pieterlange - thanks for your suggestion.
